rtos/Semaphore.h

```diff
@@ -59,9 +59,9 @@ class Semaphore {
     ~Semaphore();
 
 private:
-    osSemaphoreId_t   _id;
-    osSemaphoreAttr_t _attr;
-    os_semaphore_t    _obj_mem;
+    osSemaphoreId_t               _id;
```

Comment:
why are you indenting all the declarations? I noticed we got some files with that but not many. We don't have this covered in the coding style.

I personally don't do it as it becomes a burden after a while (like in this changeset with this new longer name variable, you have to indent all the previous declarations/definitions).